### PR TITLE
Update GEO Chinese doc

### PR DESCRIPTION
**Pull request by acelyc111**

### CODE REVIEW

**Comment by acelyc111** [comment]
https://github.com/apache/incubator-pegasus/issues/1727

File: _docs/zh/api/geo.md [hunk 1]
```diff
@@ -2,197 +2,211 @@
...
 ## 数据存储
 
-在Pegasus中，数据存储的key是hashkey+sortkey: hashkey用于数据partition，同一hashkey的数据存储在同一replica server下的一块或多块（由rocksdb实际存储的状态决定：数据随机写入后，同一hashkey下连续的sortkey空间可能分布在多个不连续的sst文件中，进行full compact后，会分布在连续sst的内）连续区域; sortkey用于在这块（或多块）区域中做数据排序的依据。
+在 Pegasus 中，数据存储的 key 是 hashkey + sortkey：hashkey 用于确定数据所处的 partition，同一 hashkey 的数据存储在同一 Replica Server 的一块逻辑连续的区域中，sortkey 用于在这块区域中做数据排序。
 
-经纬度经过坐标转换得到一维编码(字符串)后，就可以把这个一维编码作为key存储起来做**GEO索引数据**了，这里需要将这个一维编码拆分成hashkey和sortkey两部分，可以根据实际的业务场景采取不同的划分策略。
+经纬度经过坐标转换得到一维编码 CellId 后，就可以把这个一维编码作为 key 存储起来做**GEO索引数据**了，Pegasus 将这个一维编码拆分成 hashkey 和 sortkey 两部分，可以根据实际的用户场景采取不同的位数划分策略。
 
-GEO索引数据独立于原始数据，两类数据存储在不同的table内，通过[geo_client](https://github.com/apache/incubator-pegasus/blob/master/src/geo/lib/geo_client.h)做数据同步，同时支持原生Pegasus API和GEO API访问。
+GEO 索引数据独立于原始数据，两类数据存储在不同的 Pegasus 表中，通过 [GEO client](https://github.com/apache/incubator-pegasus/blob/master/src/geo/lib/geo_client.h) 做数据同步，同时支持原生 Pegasus API 和 GEO API 的访问。
```
Comment (empiredan) [inline _docs/zh/api/geo.md:84]:
```suggestion
GEO 索引数据独立于原始数据，两类数据存储在不同的 Pegasus 表中，通过 [GEO Client](https://github.com/apache/incubator-pegasus/blob/master/src/geo/lib/geo_client.h) 做数据同步，同时支持原生 Pegasus API 和 GEO API 的访问。
```

File: _docs/zh/api/geo.md [hunk 2]
```diff
@@ -2,197 +2,211 @@
...
 
-下面讨论GEO索引数据的构造方式。
+所以，在使用 Pegasus GEO 特性时，需要创建两个 Pegasus 表，一个是原始表，用于存储用户写入的原始数据，一个是 GEO 索引表，用于存储 GEO client 自动转换原始数据生成的 GEO 索引数据。
```
Comment (empiredan) [inline _docs/zh/api/geo.md:86]:
```suggestion
所以，在使用 Pegasus GEO 特性时，需要创建两个 Pegasus 表，一个是原始表，用于存储用户写入的原始数据，一个是 GEO 索引表，用于存储 GEO Client 自动转换原始数据生成的 GEO 索引数据。
```

File: _docs/zh/api/geo.md [hunk 3]
```diff
@@ -2,197 +2,211 @@
...
 
 ## 数据更新
 
 ### set
 
-`set`操作会同时更新两个table的数据: Pegasus原始数据和GEO索引数据（数据构造方式如上所述）。
+`set` 操作会同时更新上述两个表的数据，即 Pegasus 原始表数据和 GEO 索引表数据，数据构造方式也如上所述。
 
-> `set`操作的hashkey, sortkey是业务自己的格式，使用GEO API时并不做约束， 只是在geo client转存GEO索引数据时，会自动做如上所述的编码转换。
+`set`操作的 hashkey，sortkey 是用户自己的格式，使用 GEO API 时并不做约束。两个表的数据转储对用户是透明的，由 GEO client 自动完成。
```
Comment (empiredan) [inline _docs/zh/api/geo.md:138]:
```suggestion
`set`操作的 hashkey，sortkey 是用户自己的格式，使用 GEO API 时并不做约束。两个表的数据转储对用户是透明的，由 GEO Client 自动完成。
```

File: _docs/zh/api/geo.md [hunk 4]
```diff
@@ -221,19 +237,21 @@ min_level = 12
...
-目前Pegasus支持从固定格式的value中解析经纬度。经纬度以字符串形式嵌入在value中，以`|`分割， 比如:`.*|115.886447|41.269031|.*`，他们的索引由配置文件中的`latitude_index`和`longitude_index`确定。
+目前 Pegasus 支持从固定格式的 value 中解析出经纬度。经纬度以字符串形式嵌入在 value 中，以`|`分割。
+
+比如:`.*|115.886447|41.269031|.*`，经纬在 value 中的索引由配置文件中的`latitude_index`和`longitude_index`确定。
 
-## API & redis proxy
+## API & Redis Proxy
 
-Pegasus GEO特性的使用有两种方式，一是直接使用C++ geo client；二是使用redis proxy。
+Pegasus GEO 特性的使用有两种方式，一是直接使用 C++ GEO client，二是使用 Redis Proxy。
```
Comment (empiredan) [inline _docs/zh/api/geo.md:246]:
```suggestion
Pegasus GEO 特性的使用有两种方式，一是直接使用 C++ GEO Client，二是使用 Redis Proxy。
```

File: _docs/zh/api/geo.md [hunk 5]
```diff
@@ -296,23 +312,19 @@ void async_search_radial(double lat_degrees,
...
-| 半径(m) | P50(ms)     | P75(ms)     | P99(ms)     | P99.9(ms)   | 平均结果条数 | 单节点QPS |
-| ------- | ----------- | ----------- | ----------- | ----------- | ------------ | --------- |
-| 50      | 1.63071622  | 1.84607433  | 4.04545455  | 6.28        | 9.4608       | 740.287   |
-| 100     | 1.76        | 2.33614794  | 5.4         | 6.45319149  | 38.0296      | 656.66    |
-| 200     | 2.41017042  | 3.31062092  | 6.41781609  | 9.60588235  | 154.3682     | 536.624   |
-| 300     | 3.30833333  | 4.21979167  | 9.4310559   | 18          | 350.9676     | 434.491   |
-| 500     | 5.07763975  | 6.84964682  | 16.84931507 | 21.78082192 | 986.0826     | 347.231   |
-| 1000    | 12.28164727 | 18.70972532 | 43.18181818 | 57.049698   | 3947.5294    | 204.23    |
-| 2000    | 35.78666667 | 54.7300885  | 108.7331378 | 148.616578  | 15674.1198   | 98.7633   |
+| radius(m) | P50(ms)     | P75(ms)     | P99(ms)     | P99.9(ms)   | Avg result count | QPS per node |
```
Comment (empiredan) [inline _docs/zh/api/geo.md:322]:
```suggestion
| Radius(m) | P50(ms)     | P75(ms)     | P99(ms)     | P99.9(ms)   | Avg result count | QPS per node |
```

**Review by empiredan — APPROVED** [review]

**Pull request merged**